pyproject.toml

```diff
@@ -0,0 +1,28 @@
+####################### Black ########################
+#
+# Example configuration for Black.
+# https://github.com/psf/black/blob/master/pyproject.toml
+
+# NOTE: you have to use single-quoted strings in TOML for regular expressions.
+# It's the equivalent of r-strings in Python.  Multiline strings are treated as
+# verbose regular expressions by Black.  Use [ ] to denote a significant space
+# character.
+
+[tool.black]
+line-length = 88
+target-version = ['py27', 'py35', 'py36', 'py37', 'py38']
```

Comment:
We don't support Python 3.8. So you probably don't want that here.

Reply:
That's right! I will delete it.